express/properties/workflow.py

```diff
@@ -1,6 +1,178 @@
 from express.properties import BaseProperty
+import os
+import copy
+from typing import List
 
 
+# Note for naming convention: code uses snake_case, JSON uses camelCase
+
+
+# Todo: This is the quick implementation of ExabyteML, and will be depreciated eventually
+class MLQuickImplementation(BaseProperty):
+    """
+    Quick implementation of the new version of ExabyteML
+    """
+
+    def __init__(self, name, parser, *args, **kwargs):
+        """
+        Constructor for MLQuickImplementation
+
+        Args:
+            name (str): Name of the workflow
+            parser (str): Parser to use with this workflow
+        """
+        super().__init__(name, parser, *args, **kwargs)
+        self.name = name
+        self.work_dir = self.kwargs["work_dir"]
+
+        object_storage_data = self.kwargs["object_storage_data"]
+        self.obj_storage_container = object_storage_data["CONTAINER"]
+        self.obj_storage_container_region = object_storage_data["REGION"]
+        self.obj_storage_container_provider = object_storage_data["PROVIDER"]
+        self.ml_cache_dir = ".mlcache"
+        self.workflow = copy.deepcopy(self.kwargs["workflow"])
+
+    @property
+    def schema(self):
+        # Shadows the schema property in BaseProperty
+        # Not entirely sure why this is done in the old one, but the net effect is this avoids a call to
+        #   self.esse.get_property_manifest(self.name)
+        return self.esse.get_schema_by_id("workflow")
```

Comment:
1. I thought we agreed to create a parent abstract class and define schema there. Pls do that.
2. Remove the comment above from code. Github comments can be used instead/

Reply:
1. Parent class has been made, and common parts of the schema now exist there
2. Comment has been removed